README.md

```diff
@@ -101,10 +116,10 @@ Which of the following statements will work, assuming jQuery is loaded?
 
 Select all that apply:
 ```
-[] `$(".post").css("background", "peachpuff")`
-[] `$(".post").innerHTML`
+[X] `$(".post").css("background", "peachpuff")`
+[X] `$(".post").innerHTML`
 [] `$(".post").html()`
```

Comment:
`.html()` would work. `innerHTML` would work too, but it's not a jQuery method, and so shouldn't be used with a jQuery object.